packages/datadog-plugin-pg/src/index.js

```diff
@@ -17,6 +17,7 @@ class PGPlugin extends DatabasePlugin {
       kind: 'client',
       meta: {
         'db.type': 'postgres',
+        'db.process_id': processId?.toString(),
```

Comment:
Not sure what the casing convention is for tags, so ended up going with Snake Case, happy to change it to fit another existing convention.

Additionally, it seems that by leaving the tag value as a `number`, it wasn't correctly picked up in tests, even though it did show up correctly in the Datadog Trace. Thoughts?

Reply:
> Not sure what the casing convention is for tags, so ended up going with Snake Case, happy to change it to fit another existing convention.

We usually use `pid` for process IDs, so I would say let's use that unless you think it makes more sense to be more explicit. If being more explicit would be easier to understand for a `pg` user, then snake case is the way to go.

> Additionally, it seems that by leaving the tag value as a number, it wasn't correctly picked up in tests, even though it did show up correctly in the Datadog Trace. Thoughts?

Numerical tags end up in `metrics` and not in `meta`, so if the test is checking for the tag in `meta` it won't be present.

Reply:
> We usually use pid for process IDs, so I would say let's use that unless you think it makes more sense to be more explicit. If being more explicit would be easier to understand for a pg user, then snake case is the way to go.

The Postgres docs explicitly call it "Process ID", but then again, some of the utility functions that deal with PIDs do call them `pid`, so we can go that route.

> Numerical tags end up in metrics and not in meta, so if the test is checking for the tag in meta it won't be present.

Ah, good to know, will update the test and make it a number.

Reply:
@rochdev Updated!